modules/lazy-images/js/lazy-images.js

```diff
@@ -96,40 +100,50 @@ var jetpackLazyImagesModule = function( $ ) {
 	 * @param {object} image The image object.
 	 */
 	function applyImage( image ) {
-		var theImage = $( image ),
-			srcset,
+		var srcset,
 			sizes,
-			theClone;
+			imageClone,
+			lazyLoadedImageEvent;
 
-		if ( ! theImage.length ) {
+		if ( ! image instanceof HTMLImageElement ) {
 			return;
 		}
 
-		srcset = theImage.attr( 'data-lazy-srcset' );
-		sizes = theImage.attr( 'data-lazy-sizes' );
-		theClone = theImage.clone(true);
+		srcset = image.getAttribute( 'data-lazy-srcset' );
+		sizes = image.getAttribute( 'data-lazy-sizes' );
+		imageClone = image.cloneNode();
```

Comment:
**Note:** This is a potential regression.

The jQuery `clone` method would clone all custom event handlers hooked directly onto the cloned element, whereas `cloneNode` doesn't do it and it's generally not possible to clone event listeners. The only viable option that follows the existing pattern is to redefine (or rather wrap) the native `Node.addEventListener` method and keep a track of event listeners this way.

Another solution could be processing existing `img` elements in-place instead of swapping it with their clones. Direction welcome.

Reply:
I was curious as to why the image is cloned in the first place and found these:

* https://github.com/Automattic/jetpack/pull/9451
* https://github.com/Automattic/jetpack/issues/8879

I suggest we take a closer look at the originally reported issue and verify that cloning is the correct solution, because _not_ cloning might open interesting opportunities for improvement:

* It would mean no event listeners would be lost on lazy loaded images.
* It would allow more advanced placeholders to be inlined instead of 1×1 GIFs, e. g. placeholder SVGs with dimensions identical to the source image. That could solve the issue with page jumps when the image is being loaded / placeholder being replaced, because the image element would not get swapped in the DOM at all.

The latter (arguably very annoying) UX issue is currently pronounced due to the [Gutenberg Image Block not outputting the `width` and `height` image attributes](https://github.com/WordPress/gutenberg/issues/6652), which breaks the browsers' ability to layout the image ahead of time.

Reply:
@dero I've also tested and it looks good to me. 

I was just wondering if there is anyone else we should do here (maybe open another issue?), or should we ok to go?